hat/assets/js/apps/Iaso/domains/app/translations/en.json

```diff
@@ -926,6 +926,7 @@
     "iaso.workflows.editFollowUp": "Edit follow up",
     "iaso.workflows.followUps": "Follow ups",
     "iaso.workflows.mapping": "Mapping",
+    "iaso.workflows.noCondition": "Always true",
```

Comment:
Not sure about the wording. @cgerard?

Reply:
On second thought I think "No conditions" or "No condition" works better in the context of the interface